infrastructure/modules/app-cluster/main.tf

```diff
@@ -9,6 +9,7 @@ locals {
     "COGNITO_USER_POOL_ID" : var.cognito_user_pool_id,
     "RESOURCE_PREFIX" : var.resource-name-prefix,
     "COGNITO_USER_LOGIN_APP_CREDENTIALS_SECRETS_NAME" : var.cognito_user_login_app_credentials_secrets_name
+    "CUSTOM_USERNAME_REGEX" : var.custom_username_regex
```

Comment:
Needs to handle the None type and ensure if we pass None here that Python correctly picks up the None

Reply:
Have made a fix. Uses Null values which should omit the argument from the container and force a python None value when using os.environ.get